### ISSUE TITLE
Missing * in line 1670 (1.6-dev.unstable)

### ISSUE THREAD

**Issue by Lenny-Hai** [FezVrasta/ark-server-tools#480]
Hello,
i think there is an \* mssing in line 1670
 cisavedir="$(find "${arkserverroot}" -ipath "*${savedir}" | head -n1)"
instead of  cisavedir="$(find "${arkserverroot}" -ipath "${savedir}" | head -n1)"

Bye Sven

**Comment by klightspeed** [comment]
Line 1663 converts savedir to an absolute path, which is then tested case insensitively when the saved arks directory does not exist.

The `*` is only necessary for relative paths.

```
$ find "/home/steam/ARK/server1" -ipath "/home/steam/ARK/server1/ShooterGame/Saved/savedarks"
/home/steam/ARK/server1/ShooterGame/Saved/SavedArks
```

**Comment by Lenny-Hai** [comment]
But without it, it will find no directory.



Am 17.05.2016 um 01:19 schrieb Ben Peddell:

**Comment by klightspeed** [comment]
Apparently that test is not necessary anyway.  If `sAveDArkS` exists but `SavedArks` does not, the ARK server will create `SavedArks` and ignore the `sAveDArkS` directory.

**Comment by Lenny-Hai** [comment]
`root@vserver1.xxx.de ~ # /usr/local/bin/arkmanager useconfig ark2 backup
Running command 'backup' for instance 'ark2'
find: Failed to restore initial working directory: /root: Keine Berechtigung
find: Failed to restore initial working directory: /root: Keine Berechtigung
  [ ERROR  ] Saved arks directory does not exist (/home/steam/ark2.xxx.de /SavedArks)
root@vserver1.dino-paradise.de ~ # su steam
steam@vserver1:/root$ cd ..
steam@vserver1:/$ cd /home/steam
steam@vserver1:~$ /usr/local/bin/arkmanager useconfig ark2 backup
Running command 'backup' for instance 'ark2'
  [ ERROR  ] Saved arks directory does not exist (/home/steam/ark2.xxx.de /SavedArks)
steam@vserver1:~$ exit
exit
root@vserver1.xxx.de ~ #
`
Don't wonder, the echo of the directory name was for debugging only

**Comment by klightspeed** [comment]
It was a typo - `$savedrootdir` should have been `$saverootdir`

**Comment by Lenny-Hai** [comment]
Hmmm sorry, but i cannot find it....


Am 17.05.2016 um 01:53 schrieb Ben Peddell:
> - |$savedrootdir|

**Comment by klightspeed** [comment]
c8f6cee should fix the problem you've been having.

**Comment by klightspeed** [comment]
Is this fixed with 1.6-dev?
